paper/benchmarkset.tex

```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
+make the BRD4(1) system a strong candidate for the first protein hard benchmark, but a few important issues have to be addressed in order to make this assertion. They involve the flexibility of the ZA-loop in the presence and absence of 
+the substrate, and possible slow side-chain dihedral rotations near the binding site. Below we describe these challenges in more detail, and how they can be overcome. 
+
+\begin{figure*}
+\includegraphics[width=\textwidth]{figures/brd4.pdf}
+\caption{\label{fig:brd4} 
+(A) Crystal structure of BRD4(1) bound to the BZT-7 ligand (PDB ID: 3U5L), showing the four helices (blue), the ZA-loop (magenta) and the BC-loop (red). (B) Comparison of the 4LYI crystal structure in the closed state (blue), and the open state obtained after 
+350 ns of simulations of the same structure (yellow). The two hinges, Asp88 and Asp96, are shown in green. (C) The same structures of item B, but showing the side chains of the  Tyr97, Tyr139 and Asn140 residues, which remain unaffected. (D) Time series of
+the Asn140 $\chi_2$ dihedral during 350 ns of simulations of the apo BRD4(1), showing frequent and reversible transitions. 
+}
+\end{figure*}
+
+\subsubsection{Flexibility and transition of the ZA-loop}
+
+Simulations starting from the cocrystal structures of BRD4(1) bound to different ligands show that the complexes are very stable, with reported backbone RMSD values being less than 1.00 {\AA} over simulations lasting up 
+to 120ns~\cite{Su:2017:J.Biomol.Struct.Dyn.}. The same behavior was reproduced in simulations over 300 ns long (Heinzelmann et al., unpublished results), with most of the fluctuations happening in the N and C-termini, which
+are far from the ligand binding site and can be disregarded. Even though the apo crystal structures of BRD4(1) have the same conformation as the complexes~\cite{Lucas:2013:Angew.Chem.Int.Ed., Filippa:2012:Cell}, long simulations 
+of this system using different force fields have shown considerable backbone RMSD values in the ZA-loop region~\cite{Su:2017:J.Biomol.Struct.Dyn., Kuang:2015:J.Chem.Inf.Model., Steiner:2013:FEBSLett.}. However, instead of large random
+fluctuations, all cases show a sharp irreversible transition on this loop generally after 20 ns of simulations, suggesting an initial metastable state followed by stabilization of the protein backbone. 
+
+Heinzelmann et al.~\cite{Heinzelmann:2017:J.Chem.TheoryComput.} have investigated this phenomenon in more detail, with simulations and free energy calculations showing that apo crystal structures of BRD4(1) are indeed metastable, 
+and after around 50 ns of simulations a transition is seen from the so-called closed to the open state (Fig. 3B). According to the free energy calculations performed using the Amberff14SB protein force field and the TIP3P water model, 
```

Comment:
OK. So maybe we can make that more clear so it doesn't sound like you're contradicting yourself.